.coveragerc

```diff
@@ -12,4 +12,7 @@ omit =
     #src/RestrictedPython/tests/*.py
 
 [report]
-precision = 3
+precision = 2
+
+[html]
+directory = reports/coverage
```

Comment:
But this new directory structure seems only to be required if there are flake8 reports, too.